content/Objects/Objects.q.md

```diff
@@ -0,0 +1,1016 @@
+# Objects
+
++++
+
+### Part 1: Sample Code Analysis
+
+:::
+
+/// type=REPL, readonly=true
+
+```php
+<?php
+    class Person 
+    {
+        public $name = "Diana";
+        
+        public function eat()
+        {
+            echo "This is an eat() method.";
+        }
+    }
+
+    $pObject = new Person();
```

Comment:
$person is a better variable name. Can you do a find/replace? Or  $personObject would be fine if you want to also specify type